### ISSUE TITLE
Incorrect formatting of `<!DOCTYPE html>`

### ISSUE THREAD

**Issue by obnoxiousnerd** [snowpackjs/prettier-plugin-astro#1]
After running the plugin, `<!DOCTYPE html>` is formatted as `<!DOCTYPE html></!DOCTYPE>`

Minimal test case:
```astro
---

---

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>{foo}</title>
  </head>
  <body></body>
</html>

```
![image](https://user-images.githubusercontent.com/52596591/132455691-f9d07f59-305f-4cdc-a55f-cd2257f7b5f6.png)
This is only reproduced if unempty mustache expression is present.

**Comment by obnoxiousnerd** [comment]
Interesting... Now I can't replicate the issue 😕

**Comment by jasikpark** [comment]
Fixed by #7
